cmd/keymasterd/jwt.go

```diff
@@ -139,17 +139,13 @@ func (state *RuntimeState) getStorageDataFromStorageStringDataJWT(serializedToke
 		logger.Printf("err=%s", err)
 		return rvalue, err
 	}
-	//At this stage is now crypto verified, now is time to verify sane values
+	// At this stage is now crypto verified (data actually comes from a valid signer),
```

Comment:
How about "At this stage crypto has been verified"?
Also, no comma at the end of a sentence...